.github/workflows/ci.yml

```diff
@@ -8,6 +8,48 @@ env:
   GOFLAGS: -mod=vendor
 
 jobs:
+  project:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '^1.15.0'
+
+    - uses: actions/checkout@v2
+      with:
+        path: src/github.com/Microsoft/hcsshim
+        fetch-depth: 100
+
+    - name: Get Dependencies
+      shell: bash
+      env:
+        GO111MODULE: off
+      run: |
+        echo "Get dependencies"
+        go get -u -v github.com/vbatts/git-validation
+
+    - name: DCO check
+      working-directory: src/github.com/Microsoft/hcsshim
+      run: |
+        echo "DCO checks"
+
+        printf "Current working directory: " && pwd
+
+        set -x
+        if [ -z "${GITHUB_COMMIT_URL}" ]; then
+        DCO_RANGE=$(jq -r '.after + "..HEAD"' ${GITHUB_EVENT_PATH})
+        else
+        DCO_RANGE=$(curl ${GITHUB_COMMIT_URL} | jq -r '.[0].parents[0].sha + "..HEAD"')
+        fi
+
+        range=
+        commit_range="${DCO_RANGE-}"
+        [ ! -z "${commit_range}" ] && {
+          range="-range ${commit_range}"
+        }
+
+        git-validation -v ${range} -run DCO,short-subject,dangling-whitespace
```

Comment:
I kind of prefer this go in a script

Reply:
It's likely goin by the wayside but agree

Reply:
Moved to a script as the github dco app doesn't feel like showing up, so maybe we can just use this for now -_-